react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1203,10 +1203,9 @@ class Toolbox extends Component<Props, State> {
         if (this._shouldShowButton('invite')) {
             buttonsRight.push('invite');
         }
-        if (this._shouldShowButton('security') || this._shouldShowButton('info')) {
+        if ((_isModerator && this._shouldShowButton('security')) || this._shouldShowButton('info')) {
```

Comment:
looks line `info` is legacy for `security`.
```suggestion
        if (_isModerator && (this._shouldShowButton('security') || this._shouldShowButton('info')) {
```